### PR TITLE
[Bugfix/LIVE-8380] add 10s timeout to swap quotes on LLD & LLM

### PR DESCRIPTION
**Pull request by kallen-ledger**
### 📝 Description

swap service is not available
When any other error or after 10s
Then Error message: Quotes could not be generated because of technical issues. Try swapping again in a few moments. is displayed

### ❓ Context

- **Impacted projects**: `LLC, LLM, LLD` 
- **Linked resource(s)**: [LIVE-8380](https://ledgerhq.atlassian.net/browse/LIVE-8380) 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
Simulator Screenshot - iPhone 14 Pro - 2023-07-06 at 18 09 20

https://github.com/LedgerHQ/ledger-live/assets/132384348/8c842373-f52e-4690-8844-8ec0443d62b2

28-021f-4250-a4b8-1db92342e6f0)

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-5953]: https://ledgerhq.atlassian.net/browse/LIVE-5953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-8380]: https://ledgerhq.atlassian.net/browse/LIVE-8380?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sarneijim — APPROVED** [review]
is axios supporting directly these `timeout` and `timeoutErrorMessage` attributes? ! I did not know it. Great!

File: apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx [hunk 1]
```diff
@@ -137,6 +137,8 @@ const SwapForm = () => {
     onNoRates,
     ...(locationState as object),
     providers: storedProviders || undefined,
+    timeout: 10000,
```
Comment (sarneijim) [inline apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx:140]: As we are going to use this const for more calls....perhaps we could save it in a constant

**Comment by kallen-ledger** [comment]
> is axios supporting directly these `timeout` and `timeoutErrorMessage` attributes? ! I did not know it. Great!

Yes these are directly from axios!

File: apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx (hunk 1 above)
Comment (kallen-ledger) [reply apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx:140]: Good idea! 😄

**Pull request merged**